symfony/plugins/orangehrmLeavePlugin/test/fixtures/testcases/EmployeeLeaveRequestAPITestCases.yaml

```
@@ -1357,3 +1357,19 @@ GetAll:
     ]
     meta:
       total: 1
+
+  filter to check for purged employee:
+    userId: 1
+    now:
+      datetime: '2021-01-01'
+    services:
+      pim.employee_service: \OrangeHRM\Pim\Service\EmployeeService
+      leave.leave_request_service: \OrangeHRM\Leave\Service\LeaveRequestService
+    factories:
+      core.authorization.user_role_manager: [ '\OrangeHRM\Core\Authorization\Manager\UserRoleManagerFactory', 'getUserRoleManager' ]
```

```suggestion
      core.authorization.user_role_manager: [ '\OrangeHRM\Core\Authorization\Manager\UserRoleManagerFactory', 'getNewUserRoleManager' ]
```
can you refactor other places in the file

---

changed in `EmployeeLeaveRequestAPITestCases` and `LeaveAPITestCase`